nym-vpn-app/src/screens/home/new-feature-alert/streaming-optimized-label/StreamingOptimizedLabel.tsx

```diff
@@ -0,0 +1,55 @@
+import { useNavigate } from 'react-router';
+import { useTranslation } from 'react-i18next';
+import { ButtonText, Toast } from '../../../../ui/index';
+import { routes } from '../../../../router';
+import { useMainDispatch, useMainState } from '../../../../contexts/index';
+import { StateDispatch } from '../../../../types/index';
+import { setStreamOptimizedLabelSeen } from './utils';
+
+export function StreamingOptimizedLabel() {
+  const navigate = useNavigate();
+  const { streamingOptimizedLabelSeen } = useMainState();
+  const dispatch = useMainDispatch() as StateDispatch;
+  const { t } = useTranslation('notifications');
+
+  const handleClick = () => {
+    setStreamOptimizedLabelSeen(dispatch);
+    navigate(routes.exitNodeLocation);
+  };
+
+  const onOpenChange = (open: boolean) => {
+    if (!open) {
+      setStreamOptimizedLabelSeen(dispatch);
+    }
+  };
+
+  if (streamingOptimizedLabelSeen) {
+    return null;
+  }
+
+  return (
+    <Toast
```

Comment:
You should not need to use `<Toast />` component directly:  check `const { push } = useInAppNotify();` for examples on how to send such notification.
(Maybe you will need some refactoring in order to pass in a component instead of raw text)